Setup.bat

```diff
@@ -43,7 +43,7 @@ call :MarkStartOfBlock "Setup variables"
     set CORE_SDK_DIR=%BUILD_DIR%\core_sdk
     set WORKER_SDK_DIR=%~dp0\SpatialGDK\Source\Public\WorkerSdk
     set BINARIES_DIR=%~dp0\SpatialGDK\Binaries\ThirdParty\Improbable
-    set SCHEMA_COPY_DIR=%~dp0..\..\..\spatial\schema
+    set SCHEMA_COPY_DIR=%~dp0..\..\..\spatial\schema\improbable\unreal\gdk
```

Comment:
This is a driveby fix for Setup.bat as it accidentally deleted the entire schema folder when run. the schemas in the extras folder have been updated to fit as well